README.md

```diff
@@ -208,8 +208,9 @@ The following model architectures, tasks and device distributions have been vali
 <div align="center">
 
 | Architecture     | Training | Inference            | Tasks |
-|------------------|:--------:|:--------------------:|:-----:|
+|------------------|:--------:|:--------------------:|:------|
 | Llama 2          | <li>Multi card</li> |           | <li>[DPO Pipeline](https://github.com/huggingface/optimum-habana/tree/main/examples/trl)</li> |
+| Llama 2          | <li>Multi card</li> |           | <li>[PPO Pipeline](https://github.com/huggingface/optimum-habana/tree/main/examples/trl)</li> |
```

Comment:
Training should work with DeepSpeed too in theory. Maybe we can add a `:heavy_check_mark:` instead of just "Multi card"?